src/createServer.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { router: usersRouter } = require('./routers/users');
```

Comment:
Use destructuring assignment to import usersRouter and expensesRouter in a single line, like: `const { usersRouter, expensesRouter } = require('./routers');`

Reply:
I'm not quite sure how to do it since both routers are based in separate files.